scrapi/harvesters/dataone.py

```diff
@@ -150,17 +154,13 @@ def harvest(self, start_date=None, end_date=None):
         for record in records:
             # This ID is unique per data package, but won't unify multiple packages for the same project
             doc_id = record.xpath("str[@name='id']")[0].text
-            format_type = record.xpath("str[@name='formatType']")[0].text
             record = ElementTree.tostring(record, encoding=self.record_encoding)
-            if format_type.lower() != 'metadata':
-                logger.info('Not normalizing record with ID {}, type {}'.format(doc_id, format_type))
-            else:
-                xml_list.append(RawDocument({
-                    'doc': record,
-                    'source': self.short_name,
-                    'docID': copy_to_unicode(doc_id),
-                    'filetype': 'xml'
-                }))
+            xml_list.append(RawDocument({'doc': record,
+                                         'source': self.short_name,
+                                         'docID': copy_to_unicode(doc_id),
+                                         'filetype': 'xml'}
```

Comment:
Since I see you added a trailing comma to the dictionary down below, you might as well be consistent and keep the format same here.